elixir/apps/domain/lib/domain/auth/adapter/openid_connect/directory_sync.ex

```diff
@@ -382,6 +385,43 @@ defmodule Domain.Auth.Adapter.OpenIDConnect.DirectorySync do
     end)
   end
 
+  defp maybe_email_sync_error(provider) do
+    with {:ok, provider} <-
+           Domain.Auth.fetch_active_provider_by_id(provider.id, preload: :account) do
+      send_sync_error_email(provider)
+    else
+      _ ->
+        Logger.warning("Failed to fetch provider while attempting to send sync error email")
+    end
+  end
+
+  defp send_sync_error_email(provider) do
+    if sync_error_email_sent_today?(provider) do
+      Logger.info("Sync error email already sent today for Provider: #{provider.id}")
```

Comment:
This should be pretty standard log message, I think no need to make it `info` level.
```suggestion
      Logger.debug("Sync error email already sent today for Provider: #{provider.id}")
```